listenbrainz/model/playlist.py

```diff
@@ -0,0 +1,49 @@
+import uuid
+from datetime import datetime
+
+from sqlalchemy.dialects.postgresql import UUID, JSONB
+
+from listenbrainz.model import db
+from listenbrainz.webserver.admin import AdminModelView
+
+
+class Playlist(db.Model):
+    __bind_key__ = 'timescale'
+    __tablename__ = "playlist"
+    __table_args__ = {"schema": "playlist"}
+
+    id = db.Column(db.Integer, autoincrement=True, primary_key=True)
```

Comment:
just out of interest - what is the purpose of `autoincrement=True, primary_key=True` in a model definition? I assume that this is purely so that if you use sqlalchemy to create the table, that it correctly creates necessary indexes/sequences/etc. Given that we make our tables manually, what do we think about having them defined here as well. Does it make sense?

Reply:
Yeah, its unlikely we'll need those but I added them here anyways for consistency with the SQL table definition. Flask-admin also does some UI customization based on whether a column is primary_key or not but that's simple and one liner to implement if we wish to keep it.

Reply:
I think that this is something that we should come up with a development guideline for. It's nice that flask-admin takes these hints for additional functionality, so I guess it makes sense to keep it here for now.